core/src/navigation/GlobalSearch.html

```diff
@@ -122,6 +126,23 @@
     );
   });
 
+  function getSearchPlaceholder(searchProvider) {
+    if (!searchProvider || !searchProvider.inputPlaceholder) {
+      return undefined;
+    }
+    const currentLocale = LuigiI18N.getCurrentLocale();
+    if (GenericHelpers.isFunction(searchProvider.inputPlaceholder)) {
+      return searchProvider.inputPlaceholder();
+    }
+    if (typeof searchProvider.inputPlaceholder === 'string') {
+      return searchProvider.inputPlaceholder;
+    }
+    if (typeof searchProvider.inputPlaceholder === 'object') {
+      let placeholder = searchProvider.inputPlaceholder[currentLocale];
+      return placeholder;
```

Comment:
Why not just return it directly? Is `placeholder` attribute necessary?

Reply:
Done removed